### PR TITLE
Re-activate and Update Alipay.com.xml

### PR DESCRIPTION
**Pull request by cschanaj**
#10545 ~`cshall`, `my` and `openhome` WFM but failed on Travis for unknown reason. Possibly some targets are `only-available-in-china`.~

~P.S. I cannot confirm this because I am outside of the GFW.~

### CODE REVIEW

File: src/chrome/content/rules/Alipay.com.xml [hunk 1]
```diff
@@ -1,47 +1,22 @@
...
-		- huabei.alipay.com
-		- intl.alipay.com
-		- .kcart.alipay.com
-		- mobile.alipay.com
-		- payservice.alipay.com
-		- pcreditprod.alipay.com
-		- qianbao.alipay.com
-		- www.alipay.com
+		Timeout was reached:
+			 - yebprod.alipay.com
```
Comment (J0WI) [inline src/chrome/content/rules/Alipay.com.xml]: WFM

File: src/chrome/content/rules/Alipay.com.xml [hunk 2]
```diff
@@ -58,36 +33,31 @@ Fetch error: http://overseas.alipay.com/ => https://overseas.alipay.com/: (6, 'C
...
 	<target host="kuai.alipay.com" />
 	<target host="lab.alipay.com" />
 	<target host="life.alipay.com" />
 	<target host="memberprod.alipay.com" />
 	<target host="mp.alipay.com" />
-	<target host="my.alipay.com" />
-	<target host="omeo.alipay.com" />
+	<!-- <target host="my.alipay.com" /> -->
 	<target host="open.alipay.com" />
-	<target host="openhome.alipay.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Alipay.com.xml]: WFM

File: src/chrome/content/rules/Alipay.com.xml [hunk 3]
```diff
@@ -58,36 +33,31 @@ Fetch error: http://overseas.alipay.com/ => https://overseas.alipay.com/: (6, 'C
...
 	<target host="jf.alipay.com" />
 	<target host="jiaofei.alipay.com" />
 	<target host="job.alipay.com" />
-	<target host="kcart.alipay.com" />
 	<target host="kuai.alipay.com" />
 	<target host="lab.alipay.com" />
 	<target host="life.alipay.com" />
 	<target host="memberprod.alipay.com" />
 	<target host="mp.alipay.com" />
-	<target host="my.alipay.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Alipay.com.xml]: WFM

File: src/chrome/content/rules/Alipay.com.xml [hunk 4]
```diff
@@ -58,36 +33,31 @@ Fetch error: http://overseas.alipay.com/ => https://overseas.alipay.com/: (6, 'C
 	<target host="bao.alipay.com" />
 	<target host="blog.alipay.com" />
 	<target host="cmspromo.alipay.com" />
-	<target host="cshall.alipay.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Alipay.com.xml]: WFM

File: src/chrome/content/rules/Alipay.com.xml [hunk 5]
```diff
@@ -58,36 +33,31 @@ Fetch error: http://overseas.alipay.com/ => https://overseas.alipay.com/: (6, 'C
...
 	<target host="hi.alipay.com" />
-	<target host="home.alipay.com" />
 	<target host="hongbao.alipay.com" />
 	<target host="huabei.alipay.com" />
 	<target host="img.alipay.com" />
 	<target host="intl.alipay.com" />
 	<target host="jf.alipay.com" />
 	<target host="jiaofei.alipay.com" />
 	<target host="job.alipay.com" />
-	<target host="kcart.alipay.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Alipay.com.xml]: WFM

File: src/chrome/content/rules/Alipay.com.xml [hunk 6]
```diff
@@ -58,36 +33,31 @@ Fetch error: http://overseas.alipay.com/ => https://overseas.alipay.com/: (6, 'C
...
 	<target host="kuai.alipay.com" />
 	<target host="lab.alipay.com" />
 	<target host="life.alipay.com" />
 	<target host="memberprod.alipay.com" />
 	<target host="mp.alipay.com" />
-	<target host="my.alipay.com" />
-	<target host="omeo.alipay.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Alipay.com.xml]: `http://omeo.alipay.com/service/checkcode?sessionID=0`

**Comment by cschanaj** [comment]
@J0WI fixed accordingly. thanks!!

**Pull request merged**